dask/tests/test_distributed.py

```diff
@@ -259,7 +259,7 @@ async def f():
         z = await y.persist()
         assert len(z.dask) == 1
 
-    asyncio.get_event_loop().run_until_complete(f())
+    asyncio.run(f())
```

Comment:
Was looking at using `asyncio.run`, but couldn't tell if there was something I was missing about how the event loop is setup/used here. Glad to see it is that simple 🙂